.travis.yml

```diff
@@ -3,6 +3,10 @@ os:
   - linux
 compiler:
   - gcc
+addons:
+  apt:
+    packages:
+    - python3-venv
```

Comment:
Do we have internal systems that will need to be updated too?

Reply:
yes, I will describe in separate communication